backend/app/Http/Controllers/TransactionController.php

```diff
@@ -25,4 +26,88 @@ public function show(Transaction $transaction)
+    {
+        function processAccounts($payload)
+        {
+
+            $myAccount = Account::findOrFail($payload['account_id']);
+            $myLatestTransaction = getLatestTransaction($myAccount->id);
+
+            if ($payload['transaction_type'] !== 'TRANSFER') {
+
+                return [
+                    [
+                        'account_id' => $myAccount->id,
+                        'user_id' => $myAccount->user_id,
+                        'starting_balance' => getStartingBalance($myLatestTransaction),
+                    ],
+                ];
+            }
+
+            $recAccount = Account::findOrFail($payload['receiver_id']);
+            $recLatestTransaction = getLatestTransaction($recAccount->id);
+
+            return [
+                [
+                    'account_id' => $myAccount->id,
+                    'user_id' => $myAccount->user_id,
+                    'starting_balance' => getStartingBalance($myLatestTransaction),
+                ],
+                [
+                    'account_id' => $recAccount->id,
+                    'user_id' => $recAccount->user_id,
+                    'starting_balance' => getStartingBalance($recLatestTransaction),
+                ],
+            ];
+
+        }
+
+        //START
+
+        $payload = $request->collect();
+
+        $accounts = processAccounts($payload);
+
+        if ($payload['transaction_type'] === 'CREDIT') {
+            //TODO: Insert Credit Code
+        }
+
+        if ($payload['transaction_type'] === 'DEPT') {
+            //TODO: Insert Deposit Code
+        }
+
+        if ($payload['transaction_type'] === 'TRANSFER') {
+            $transaction_date = now();
+            $senderData = [
+                'account_id' => $accounts[0]['account_id'],
+                'user_id' => $accounts[0]['user_id'],
+                'transaction_type' => $payload['transaction_type'],
+                'category' => 'SENDER',
+                'description' => $payload['description'],
+                'starting_balance' => $accounts[0]['starting_balance'],
+                'transaction_amount' => $payload['amount'] * -1,
+                'transaction_date' => $transaction_date,
+            ];
+            $receiverData = [
+                'account_id' => $accounts[1]['account_id'],
+                'user_id' => $accounts[1]['user_id'],
+                'transaction_type' => $payload['transaction_type'],
+                'category' => 'RECIPIENT',
+                'description' => $payload['description'],
+                'starting_balance' => $accounts[1]['starting_balance'],
+                'transaction_amount' => $payload['amount'],
+                'transaction_date' => $transaction_date,
+            ];
+
+            $sendRow = Transaction::create($senderData);
+            $receiverData['transaction_id'] = $sendRow->id;
+            Transaction::create($receiverData);
+
+            return response()->json([
```

Comment:
turn to resourceful response TransactionCreateResource

Reply:
Something like this? `return ( new Transaction($receiverData))->response()->setStatusCode(200);`